defaults/main.yml

```diff
@@ -1,9 +1,59 @@
-# SPDX-License-Identifier: MIT
 ---
 # Here is the right place to put the role's input variables.
 # This file also serves as a documentation for such a variables.
 
+## Required vars
 # Realm name to join
 ad_integration_realm: ""
 # Password of the user used to authenticate the join operation
 ad_integration_password: ""
+
+## Optional vars
+# Active Directory user used to join the domain
+ad_integration_user: "Administrator"
+
+# Active Directory domain controller's host name or IP address to join via that domain controller directly.
+ad_integration_join_to_dc: null
+
+# Turn it off to use UID and GID information stored in the directory (as-per RFC2307) rather than automatically generating UID and GID numbers
```

Comment:
```suggestion
# Turn it off to use UID and GID information stored in the directory (as-per RFC2307)
# rather than automatically generating UID and GID numbers
```